src/compat/object/pickBy.spec.ts

```diff
@@ -119,23 +189,6 @@ describe('pickBy', () => {
     expect(argsList).toEqual(expected);
   });
 
-  it(`iterates over own string keyed properties of objects`, () => {
```

Comment:
Oh, isn't this test case in lodash?

Reply:
https://github.com/lodash/lodash/blob/v5-wip/test%2Fiteration-methods.spec.js#L237-L237

Reply:
pickby was excluded (forInMethods)